Client/Frontend/Browser/URLBarView.swift

```diff
@@ -148,6 +133,26 @@ class URLBarView: UIView, BrowserLocationViewDelegate, AutocompleteTextFieldDele
         finishEditingAnimation(false)
     }
 
+    private func makeTabsButton(count: Int) -> UIButton {
+        let tabsButton = InsetButton()
+        tabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
```

Comment:
Yeah, I was wondering what this was about too (this is just existing code we had moved into this function). I jiggled some code around and removed it.